qemu/tests/virtio_fs_share_data.py

```diff
@@ -340,6 +346,7 @@ def get_stdev(file):
                     test.fail("st_dev are the same on diffrent device.")
         finally:
             if os_type == "linux":
+                session.cmd("cd /")
```

Comment:
Sorry, if we use `safe_rmdir()` we don't need to return to the root directory, you can see that it takes into account the problems we occasionally encountered before. 
https://github.com/avocado-framework/avocado-vt/blob/master/virttest/utils_misc.py#L892

Reply:
hi, we keep this as umount need to return to other dir. thanks